src/Components/CategoryInput.tsx

```diff
@@ -11,20 +11,26 @@ interface Props {
 }
 
 export const CategoryInput = ({ getCategory }: Props) => {
-  const inputRef = useRef<string>('')
-  //Should get category data from db
+  const [inputCat, setInputCat] = useState<string>('')
   const [categories, setCategories] = useState<string[]>(['Common misconception', 'Form similar to answer'])
```

Comment:
These values are dummy values currently and will be fetched from the server, right?

Reply:
Yep the categories will be replaced, but the inputCat is not.